site/Makefile

```diff
@@ -1,28 +1,22 @@
 HUGO?=hugo
-# the officially recommended unofficial docker image
 HUGO_IMG?=hugomods/hugo:0.115.3
 
 THEME_MODULE = github.com/nginxinc/nginx-hugo-theme
-## Pulls the current theme version from the Netlify settings
-THEME_VERSION = $(NGINX_THEME_VERSION)
-NETLIFY_DEPLOY_URL = ${DEPLOY_PRIME_URL}
+THEME_VERSION = 0.41.14
```

Comment:
we can use renovate to keep these versions up to date if it helps

Reply:
I'll keep that in mind! Right now the docs action is automatically manages both the deployment of the website and the version of the theme to use, so this only affects local development.

I plan to build a minimal custom Docker image with all of the docs-related tooling contained within, so the Makefile could end up looking quite different.